api/pkg/auth/service/auth.go

```diff
@@ -62,6 +62,20 @@ type Service interface {
 	HubAuthenticate(res http.ResponseWriter, req *http.Request)
 }
 
+type cookie struct {
```

Comment:
Can you move this struct to [app.go](https://github.com/tektoncd/hub/blob/main/api/pkg/auth/app/app.go)